samples/kafka.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "apache-kafka",
+  "category": "streaming",
+  "namespace": "default",
+  "image": {
+    "provisioners": [
+      {
+        "name": "apache-kafka-image",
+        "type": "docker",
```

Comment:
seems this should be `shell` type as what the inline commands not that specific for docker, and we don't support docker type in the backends for now :(